src/java.base/share/classes/java/util/Locale.java

```diff
@@ -534,28 +534,9 @@
  * {@code in}. Since Java SE 17, this is no longer the case. Each
```

Comment:
```
 * <p>Locale's constructors have always converted three language codes to
 * their earlier, obsoleted forms: {@code he} maps to {@code iw},
 * {@code yi} maps to {@code ji}, and {@code id} maps to
 * {@code in}. Since Java SE 17, this is no longer the case.
```

This history was relevant when the property existed. Since this is no longer the case, and we're quite a few releases away from 17, can we also remove this wording as well. Users on 26 should only be concerned with the "old" to "modern" mapping.